rpc/src/optimistically_confirmed_bank_tracker.rs

```diff
@@ -102,7 +140,8 @@ impl OptimisticallyConfirmedBankTracker {
         pending_optimistically_confirmed_banks: &mut HashSet<Slot>,
         last_notified_confirmed_slot: &mut Slot,
         highest_confirmed_slot: &mut Slot,
-        bank_notification_subscribers: &Option<Arc<RwLock<Vec<BankNotificationSender>>>>,
+        highest_root_slot: &mut Slot,
+        bank_notification_subscribers: &Option<Arc<RwLock<Vec<SlotNotificationSender>>>>,
```

Comment:
nit: rename this and all the other references to `bank_notification_subscribers` to `slot_notification_subscribers`

Reply:
Done